integration/run_test.go

```diff
@@ -520,3 +521,198 @@ func TestRunInNamespace(t *testing.T) {
 		}
 	}
 }
+
+func TestRunWithIdMap(t *testing.T) {
```

Comment:
Should this also test that the ids are mapped correctly inside the container?

Reply:
Yeah, good point, in some manual tests that I ran the id mapping would fail because the container wouldn't recognize the IDs inside. Added tests for this.